### DISCUSSION TITLE
Addition of webcams?

### DISCUSSION THREAD

**Discussion by drpimples** [iklein99/homebridge-smartthings#155] [Ideas]
Would you consider the addition of webcams like the Wyze cams?  They have an API but no good plugins for Homebridge.

**Comment by kshartman** [comment]
Why add it to this plugin when scrypted is a platform built for it.  Wyze is supported:  https://www.reddit.com/r/Scrypted/comments/18ogvre/scrypted_now_has_a_wyze_camera_plugin/?rdt=62399
